ompi/mca/coll/libnbc/libdict/hb_tree.h

```diff
@@ -19,43 +20,20 @@ typedef struct hb_tree hb_tree;
 
 hb_tree *hb_tree_new __P((dict_cmp_func key_cmp, dict_del_func key_del,
 						  dict_del_func dat_del));
-dict	*hb_dict_new __P((dict_cmp_func key_cmp, dict_del_func key_del,
-						  dict_del_func dat_del));
-void	 hb_tree_destroy __P((hb_tree *tree, int del));
 
 int hb_tree_insert __P((hb_tree *tree, void *key, void *dat, int overwrite));
-int hb_tree_probe __P((hb_tree *tree, void *key, void **dat));
 void *hb_tree_search __P((hb_tree *tree, const void *key));
 int hb_tree_remove __P((hb_tree *tree, const void *key, int del));
-void hb_tree_empty __P((hb_tree *tree, int del));
-void hb_tree_walk __P((hb_tree *tree, dict_vis_func visit));
-unsigned hb_tree_count __P((const hb_tree *tree));
-unsigned hb_tree_height __P((const hb_tree *tree));
-unsigned hb_tree_mheight __P((const hb_tree *tree));
-unsigned hb_tree_pathlen __P((const hb_tree *tree));
-const void *hb_tree_min __P((const hb_tree *tree));
-const void *hb_tree_max __P((const hb_tree *tree));
 
 struct hb_itor;
 typedef struct hb_itor hb_itor;
 
 hb_itor *hb_itor_new __P((hb_tree *tree));
-dict_itor *hb_dict_itor_new __P((hb_tree *tree));
 void hb_itor_destroy __P((hb_itor *tree));
 
 int hb_itor_valid __P((const hb_itor *itor));
-void hb_itor_invalidate __P((hb_itor *itor));
 int hb_itor_next __P((hb_itor *itor));
```

Comment:
What about this symbol? There are many more here that don't have proper naming but seem to be global symbols.

Reply:
hb_itor_invalidate can be made static since it's only used in a single file. Some of the hb_ symbols are used in inline code blocks in nbc_internal.h and get compiled into more than one file, so can't be made static. I'm currently working on a second pull request where any of the global symbols in my list used in two or more files are being renamed to ompi_ or opal_ prefixes to conform to naming conventions. I might have that pull request done on Monday. 

A final step if completing the 2nd pull request is re-running the test program to make sure I haven't missed anything before moving on to treematch.

Then there is the 3rdparty/treematch code which has a lot of symbols to be fixed, as a 3rd pull request.